modules/persistent-storage-csi-manila-limitations.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-manila.adoc
+//
+:_content-type: CONCEPT
+
+[id="persistent-storage-csi-manila-limitations_{context}"]
+= Manila CSI Driver Operator limitations
+
+The following limitations apply to the Manila Container Storage Interface (CSI) Driver Operator:
+
+* Only NFS is supported
++
+OpenStack Manila supports many Network Attached Storage protocols, such as NFS, CIFS, and CEPHFS, and these can be selectively enabled in the OpenStack cloud. The Manila CSI Driver Operator in {product-title} only supports using the NFS protocol. If NFS is not available and enabled in the underlying OpenStack cloud, you cannot use the Manila CSI Driver Operator to provision storage for {product-title}.
+
+* KCS and scale warning
++
+In Red Hat OpenStack Platform 16.1, the Shared File Systems service (Manila) with CephFS through NFS fully supports serving shares to {product-title} through the Manila CSI. However, this solution is not intended for massive scale. Be sure to review important recommendations at https://access.redhat.com/articles/6667651.
+
+* Snapshots are not supported if the back end is CephFS-NFS
++
+To take snapshots of persistent volumes and revert volumes to snapshots, you must ensure that the Manila share type that you are using supports these features. A Red Hat OpenStack administrator must enable support for snapshots (`share type extra-spec snapshot_support`) and for creating shares from snapshots (`share type extra-spec create_share_from_snapshot_support`) in the share type associated with the storage class you intend to use.
```

Comment:
Style guide discourages parentheses, but I don't have any brilliant alternatives to suggest here :sweat_smile: Mentioning in case you come up with something :slightly_smiling_face: